MiniApp/Classes/Extensions/MiniApp+URLRequest.swift

```diff
@@ -0,0 +1,9 @@
+extension URLRequest {
+    static func createURLRequest(url: URL, environment: Environment) -> URLRequest {
```

Comment:
Are there any unit tests for this?

Reply:
Xcode was showing that the code coverage as 100% for this extension but there was no dedicated testcase written for this file. Surprisingly testcases were passed too. Now I have created a Unit testcases for createURLRequest

Reply:
It can be misleading to rely on the coverage percentage to determine if a function/class/whatever has got specific tests to verify its behaviour.

Reply:
Noted